test/quantized_ops/test_quantized_matmul.py

```diff
@@ -97,9 +106,90 @@ def test_q_linear_hlo(self):
 
       output = torch.ops.xla.quantized_matmul(x, w_int, scaler)
       hlo = torch_xla._XLAC._get_xla_tensors_hlo([output])
-      print(hlo)
       self.assertTrue(re.search(r'bf16.*dot.*bf16.*s8', hlo) is not None)
 
+  def test_int4_pack_unpack(self):
+    x = torch.randint(-8, 7, (2, 12), dtype=torch.int8)
+    orig_dtypes = [
+        torch.int8,
+    ]
+    for dtype in orig_dtypes:
+      data = x.to(dtype)  # make sure the tensor is within the range of dtype.
+      packed = pack_4bit(data, dtype=dtype)
```

Comment:
so data was in int8, what does `pack_4bit ` exactually do?

Reply:
same, we don't need pack/unpack now. I put a sumamry of the flow in PR description